cloudprint/cloudprint.py

```diff
@@ -209,6 +209,8 @@ class CloudPrintProxy(object):
     def __init__(self, auth):
         self.auth = auth
         self.sleeptime = 0
+        self.site = ""
+        self.prefix = False
```

Comment:
Can we use single quotes here for site. We should also be able to use remove the `prefix` and use the fact that an empty string is falsey.

The tests below can be changed from
```
if self.prefix:
```
to 

```
if self.site:
```